source/computer/commands.rst

```diff
@@ -63,54 +62,556 @@ Linux/macOS 下有成百上千个命令，每个命令都有众多选项。这
 
     -   其他
 
+        - `df`_
+        - `du`_
         - `ssh`_
         - `scp`_
         - `rsync`_
+        - `wget`_
+        - `top`_
 
 ----
 
 cat
 ---
 
+``cat`` 命令名来源于 con\ **cat**\ enate（即“拼接”）。该命令可以输出某个文件的
+内容，或将多个文件\ **拼接**\ 起来。
+
+将一个文件的内容打印到标准输出::
+
+    $ cat file
+
+将文件 :file:`file1` 和 :file:`file2` 合并到一个文件 :file:`target_file` 中::
+
+    $ cat file1 file2 > target_file
+
+打印文件内容并显示行号::
+
+    $ cat -n file
```

Comment:
Remove `cat -n`?
```suggestion
```

Reply:
Not really. If we use the above way, we can keep it.

Reply:
Keep it in f954636.

Reply:
I agree.